test/integration/default/controls/tests.rb

```diff
@@ -0,0 +1,7 @@
+include_controls 'linux-baseline' do
+  # skip entropy test, as our short living test VMs usually do not
+  # have enough
+  skip_control 'os-08'
```

Comment:
Just as an alternative: I install the `haveged` package on all my systems to feed the random devices ... would it be possible to start this daemon also in the testing vm's?

Reply:
@mcgege should we make it for testing only? Or should it be a general part of implementations?

Reply:
Well, I think it won't harm to implement it as a general solution, but there might be other ways to solve this out there. This is just my recommendation to be compliant here ...
Could this be easily implemented for testing?

Reply:
I tried to go with haveged way: its possible, however the code isn't really two lines of code. For CentOS `haveged` is located within EPEL, so we need to install `epel-release` on RH family (and only on RH, so its an additional check) and so on. I personally would like to keep the `skip_control` way, as the `haveged` implementation is much more bigger. Is it okay for you?

Reply:
Of course it is!